README.md

```diff
@@ -1,66 +1,123 @@
+### Q: Will I be able to import my data from an official DMM server?
+Theoretically, yes.   
+The only downside is it will require your API key to import your ships2 data and your admiral information. This may get your account banned for whatever lame DMM reason.  
+It is recommended you start fresh on your copy of this server. 
 
-* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/) that's insanely unwieldly, performs about as well as a football team in Antarctica and is a nightmare to maintain.  
-  I've written large applications in PHP before, and I'm not going there again.
+### Q: What is the recommended requirements?
+- Debian or Ubuntu based distro. Fedora and friends may work, but please, YMMV.
+- Python 3.x and knowledge about virtualenv (setup and activate).
+- Internet connection to download the DMM assets.
 
-* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well, preferably without me having me hire another server I can't pay for.  
-  I run a JIRA (Java) and a Discourse (Ruby) instance, and they both chew up nearly everything their servers have to give.
+Protip: Use a VM (VMWare, VirtualBox). If you mess up, simply hose it and start squeaky clean. Or you can use an ARM based linux box (R-Pi, ODROID, etc).
 
+### Q: Why is it built on Python?
+* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/).
+* Ruby / Java are both massive resource hogs, and this server needs to be able to scale well.  
+  They seem to eat the server alive.
 * NodeJS is actually a good candidate, mostly thanks to being blazing fast.  
-  Unfortunately, nearly everything around it (including the runtime itself) is in beta, and many things (such as database abstraction layers) simply aren't mature enough for me to want to depend on it - I have yet to see a robust migration system, for instance.  
-  One day I might even port over at least parts of the server to it (the game API comes to mind), but not right now.
+  Unfortunately, nearly everything around it (including the runtime itself) is in beta.
 
-Python isn't the fastest thing around, but both it and its ecosystem are very stable and mature. SQLAlchemy + Alembic alone makes an incredibly robust database abstraction layer, that nothing I've seen in NodeJS can match. (Do feel free to prove me wrong though.)
+Python isn't the fastest thing around, but both it and its ecosystem are very stable and mature. SQLAlchemy + Alembic alone makes an incredibly robust database abstraction layer, that nothing I've seen in NodeJS can match. 
 
+### Q: Why PostgreSQL?
+* ***MySQL*** is [notoriously awful](http://grimoire.ca/mysql/choose-something-else), with the most important piece of incompetent nonsense being the lack of Transactional DDL… which means a failed migration (for whatever reason) may leave the database in an inconsistent state.
```

Comment:
Oops, looks like an ellipsis character (`…`) slipped in here. Looks weird in monospace, so you may want to make it three dots.